modules/n1ql/pages/n1ql-language-reference/transactions.adoc

```diff
@@ -139,6 +139,8 @@ These are explained in the sections below.
 Note that some Couchbase SDKs provide APIs to support Couchbase transactions.
 For further details, refer to {transactions}[Transactions].
 
+TIP: For a walkthrough of how to create a transaction and use other transaction statements with the Query Workbench, cbq shell, or Query REST API, see xref:n1ql:n1ql-language-reference/transactions-walkthrough.adoc[].
+
```

Comment:
Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)